frontend/src/pages/my-projects/components/create-project-form/create-project-form.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Input } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useAppForm, useAppSelector, useCallback } from '~/libs/hooks/hooks.js';
+import {
+  type ProjectCreateRequestDto,
+  projectCreateValidationSchema,
+} from '~/packages/projects/projects.js';
+import { type UserAuthResponse } from '~/packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onSubmit: (payload: ProjectCreateRequestDto) => void;
+  className?: string | undefined;
+};
+
+const CreateProjectForm: React.FC<Properties> = ({
+  onSubmit,
+  className = '',
+}: Properties) => {
+  const user = useAppSelector(({ auth }) => {
+    return auth.user as UserAuthResponse;
+  });
+
+  const { control, errors, handleSubmit } = useAppForm<ProjectCreateRequestDto>(
+    {
+      defaultValues: {
+        name: '',
+        userId: user.id,
+      },
+      validationSchema: projectCreateValidationSchema,
+    },
+  );
+
+  const handleFormSubmit = useCallback(
+    (event_: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event_);
+    },
+    [handleSubmit, onSubmit],
+  );
+
+  return (
+    <div className={getValidClassNames(styles['form-wrapper'], className)}>
+      <h2>What is your business’s name?</h2>
```

Comment:
```suggestion
      <h2>Create a new business </h2>
```
`name` should be a label